### ISSUE TITLE
Package can't be installed with npm because of pre-install hook

### ISSUE THREAD

**Issue by fiws** [juliencrn/usehooks-ts#420]
### Describe the problem

```
❯ npm i usehooks-ts
npm ERR! code 1
npm ERR! path /tmp/test/node_modules/usehooks-ts
npm ERR! command failed
npm ERR! command sh -c npx only-allow pnpm
npm ERR! ╔═════════════════════════════════════════════════════════════╗
npm ERR! ║                                                             ║
npm ERR! ║   Use "pnpm install" for installation in this project.      ║
npm ERR! ║                                                             ║
npm ERR! ║   If you don't have pnpm, install it via "npm i -g pnpm".   ║
npm ERR! ║   For more details, go to https://pnpm.js.org/              ║
npm ERR! ║                                                             ║
npm ERR! ╚═════════════════════════════════════════════════════════════╝
npm ERR! npx: installed 2 in 0.87s
```

## Versions

```
❯ npm version
{
  test: '1.0.0',
  npm: '10.2.5',
  node: '21.5.0',
  acorn: '8.11.2',
  ada: '2.7.4',
  ares: '1.24.0',
  base64: '0.5.1',
  brotli: '1.1.0',
  cjs_module_lexer: '1.2.2',
  cldr: '44.1',
  icu: '74.2',
  llhttp: '9.1.3',
  modules: '120',
  napi: '9',
  nghttp2: '1.58.0',
  openssl: '3.2.0',
  simdjson: '3.6.2',
  simdutf: '4.0.4',
  tz: '2023c',
  undici: '5.28.2',
  unicode: '15.1',
  uv: '1.47.0',
  uvwasi: '0.0.19',
  v8: '11.8.172.17-node.18',
  zlib: '1.3'
}
```

### Additional context

This issue is supposed to be a bug report, not about documentation. I can not click the bug report template … it is bugged.

**Comment by denis-g** [comment]
I also have an error with an old project and a new package.

**Comment by timoisalive** [comment]
Same issue here. 😢

**Comment by mschorsch** [comment]
+1

**Comment by pixelRena** [comment]
This is also happening for me as well

**Comment by CamdenFoucht** [comment]
Same here:

```
npm ERR! code 127
npm ERR! command failed
npm ERR! command sh -c npx only-allow pnpm
npm ERR! npm WARN exec The following package was not found and will be installed: only-allow@1.2.1
npm ERR! sh: only-allow: command not found
```

**Comment by Fhasghrge** [comment]
Same issuse, beacause i build it online, never notice the online terminal, the terminal will always waiting for use input for long  time until the task be canceled, the online build task will fail

**Comment by shahkashani** [comment]
+1

**Comment by juliencrn** [comment]
I'm weirdly not able to reproduce the issue, ever uninstalling pnpm and testing on a fresh node@21 installation.
I've removed the preinstall script (`"preinstall": "npx only-allow pnpm",`) and published a new version `2.9.3`. 
Is it fixed for you?

**Comment by wadehammes** [comment]
@juliencrn getting this with latest version:

<img width="1498" alt="image" src="https://github.com/juliencrn/usehooks-ts/assets/222170/71c1848c-892c-4a25-bc00-85e1a4289ce8">

**Comment by juliencrn** [comment]
Fixed in 2.9.4
